jmbitcoin/jmbitcoin/secp256k1_ecies.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/python
+from __future__ import (absolute_import, division,
+                        print_function, unicode_literals)
```

Comment:
If bitcointx is a dependency, and it requires python >= 3.6, `from __future__ import` is not needed, because it is only relevant for python2